client/src/components/ItemsForm.jsx

```diff
@@ -1,4 +1,4 @@
-import { categoriesOut, categoriesIn } from "../lib/catergoryData.js";
+import { categoriesOut, categoriesIn } from "../lib/catergory-data.js";
```

Comment:
You can import with a `.js` extension? That's usually complained when I've tried it.